package.json

```diff
@@ -101,14 +101,14 @@
     "moment": "^2.24.0",
     "moment-timezone": "^0.5.27",
     "morgan": "^1.9.1",
-    "ora": "^4.0.2",
+    "ora": "^4.0.3",
     "path-to-regexp": "^3.1.0",
     "polyfill-crypto.getrandomvalues": "1.0.0",
     "ramda": "^0.26.1",
-    "react": "^16.11.0",
-    "react-dom": "^16.11.0",
+    "react": "^16.12.0",
```

Comment:
ld pull out react updates into a separate PR, we want to ensure that Psammead is up to date on react first before updating the version in Simorgh

Reply:
[Done](https://github.com/bbc/simorgh/pull/4712)